_includes/world/2024/tito-widget.html

```diff
@@ -2,6 +2,11 @@
     <h3 class="ticketTitle">Tickets</h3>
     <div class="iframeContainer">
         <div id="tito-widget"></div>
+        <div class="soldOutContainer">
+            <p>
+                Sold out. If tickets are returned or released we will release another batch later this year.
+            </p>
+        </div>
```

Comment:
Added this in page for two reasons, 

1 - To get rid of tito when we don't need it to avoid the "hack" from last year. 
2 - It took a bit to load with tito, now it's instant and it doesn't load after the page does.